recipes/asio-grpc/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+cmake_minimum_required(VERSION 3.14)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(asio-grpc REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} PRIVATE asio-grpc::asio-grpc)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
+
+# See https://github.com/chriskohlhoff/asio/issues/955
+target_compile_definitions(${PROJECT_NAME}
+                           PRIVATE BOOST_ASIO_DISABLE_STD_ALIGNED_ALLOC)
+
+if(CMAKE_CROSSCOMPILING)
```

Comment:
Not needed as test package never is executed when cross-building.

Reply:
But `asio_grpc_protobuf_generate` executes protoc which is why this build didn't succeed:

Failure in build 5 (e3d1a9eb809a5b717427066715e9e8885dd256bb): [All logs](https://c3i.jfrog.io/c3i/misc/summary.html?json=https://c3i.jfrog.io/c3i/misc/logs/pr/11437/5-configs/macos-m1-clang/asio-grpc/1.7.0//summary.json)